mirgecom/diffusion.py

```diff
@@ -169,7 +184,13 @@ def diffusion_facial_flux_harmonic(
     flux_without_penalty = np.dot(flux_tpair.avg, normal)
 
     # TODO: Verify that this is the correct form for the penalty term
-    tau = penalty_amount*kappa_harmonic_mean/lengthscales_tpair.avg
+    if isinstance(kappa_harmonic_mean, np.ndarray):
+        # if anisotropic, get the normal diffusivity absolute value
+        actx = normal[0].array_context
+        kappa_mean_normal = actx.np.abs(np.dot(kappa_harmonic_mean, normal))
```

Comment:
When computing the harmonic mean of vector/tensor-valued `kappa`s, should we be taking the harmonic mean first and then dotting it with the normal, or dotting each with the normal and then taking the harmonic mean? (Or are the two equivalent? TBD)

Reply:
As we discussed offline, the normal dotting is the last step. I worked on the docs (https://github.com/illinois-ceesd/mirgecom/pull/976/commits/29d3bc2c873163e9294ffe2a9e319477931b4074) to highlight the steps we are taking to compute this case. It seems that the general anisotropic case would need some thinking, but we wont have those cases, only orthotropic materials in which the thermal conductivity is a diagonal matrix, which is easily invertible.